pkg/node/node.go

```diff
@@ -254,6 +255,20 @@ func NewBee(interrupt chan os.Signal, addr string, publicKey *ecdsa.PublicKey, s
 
 	chainEnabled := isChainEnabled(o, logger)
 
+	var batchStore postage.Storer = bsm.NoOp()
```

Comment:
So I dont think we should be using mock code in production. This should be for testing only.

It would be better to define a Noop batchstore here or in a new package in postage.

Reply:
moved to a new package